gravity-forms/gw-first-error-focus.php

```diff
@@ -31,11 +31,16 @@ function gw_first_error_focus_script() {
 					// We need to reset our flag so that we can still do our focus action when the form conditional logic
 					// has been re-evaluated.
 					window['gwfef'] = false;
+					var y = 0;
+					// For a multiple page form after page 1, we need to render on the gform wrapper
+					if ($('.gf_step_current_page') && $('.gf_step_current_page').text() > 1) {
+						y = $('.gform_wrapper').offset().top;
+					}
 					var hasError = gwFirstErrorFocus().length;
```

Comment:
I'd create a new variable below `hasError` named `onSubsequentPage` or something like that...

`var onSubsequentPage = $('.gf_step_current_page') && $('.gf_step_current_page').text() > 1;`

Then, change `if (!hasError) {` to `if (!hasError && onSubsequentPage) {`

All other modifications aren't needed, I don't think.

The one caveat with this is if they navigate from the second page to the first, it won't work, but I can't think of a quick way to handle that.